gatsby-config.js

```diff
@@ -199,6 +200,28 @@ const plugins = [
   }
 ]
 
+if (process.env.GITHUB_TOKEN) {
```

Comment:
if we don't show it by default - do we still need this build-time block?

Reply:
Yes, we do. Basically, the original plugin throws if invoked without a token and this gets around that.

Reply:
👍